app/src/shared/components/Tile/index.jsx

```diff
@@ -251,24 +368,41 @@ const ProductTile = ({
                 {actions}
             </Menu>
         )}
+        <Tile.ImageContainer>
+            <Link
+                to={product.id && routes.products.edit({
+                    id: product.id,
+                })}
+            >
+                <Tile.ImageContainer autoSize>
+                    <Tile.Thumbnail src={product.imageUrl || ''} />
+                </Tile.ImageContainer>
+            </Link>
+            {!!showDataUnionBadge && (
+                <DataUnionBadge top left />
+            )}
+            {typeof numMembers !== 'undefined' && !showDeployingBadge && (
+                <IconBadge
```

Comment:
There are 2 other member count badges around here (purchase tile, marketplace product tile) but only this one links to the stats. Others, in general, link to product pages in the marketplace. Not sure if marketplace product pages (should) have access to the states. cc @juhah @mattatgit

Reply:
@mondoreale Yep, as we discussed, it would make sense for the member count badges on *purchased* DU products should link to the Product detail page > Overview section